src/main/java/dev/lambdaurora/lovely_snails/entity/SnailEntity.java

```diff
@@ -423,24 +390,21 @@ public ItemStack getChest(int slot) {
 	}
 
 	public int getInventorySize() {
-		return 50;
+		return 48;
```

Comment:
A bit confused by this change, did they separate the whole saddle/carpet stuff from normal inventory in the latest versions?